### PR TITLE
Tickets/preops 1246

### PR DESCRIPTION
**Pull request by Madejski**

### CODE REVIEW

File: tutorials-examples/portal-4.rst
Comment (ChristinaAdair) [inline tutorials-examples/portal-4.rst:1]: A1.4 "You should leave the "Output Column"... maybe explain in the large table to the right?

Comment (ChristinaAdair) [inline tutorials-examples/portal-4.rst:1]: A2.1 "an" should be "a"

Comment (ChristinaAdair) [inline tutorials-examples/portal-4.rst:1]: A2.2 "bottom" has an extra letter

Comment (ChristinaAdair) [inline tutorials-examples/portal-4.rst:1]: A3.1 you put "100" in the text, but the image shows "50"

**Review by ChristinaAdair — CHANGES_REQUESTED** [review]
Only got through Part A

File: tutorials-examples/portal-4.rst
Comment (ChristinaAdair) [inline tutorials-examples/portal-4.rst:1]: B1.1 "simple" typo

Comment (ChristinaAdair) [inline tutorials-examples/portal-4.rst:1]: B2.1 "To" typo

Comment (ChristinaAdair) [inline tutorials-examples/portal-4.rst:1]: B3.1, maybe include a picture with the "log" option selected in chart options section?

**Review by ChristinaAdair — CHANGES_REQUESTED** [review]
reviewed part A and B, some comments and suggestions for changes

File: tutorials-examples/portal-4.rst
[the remark this answers is not in the document]
Comment (Madejski) [reply tutorials-examples/portal-4.rst:1]: done

Comment (Madejski) [reply tutorials-examples/portal-4.rst:1]: done

[the remark this answers is not in the document]
Comment (Madejski) [reply tutorials-examples/portal-4.rst:1]: done

Comment (Madejski) [reply tutorials-examples/portal-4.rst:1]: done

Comment (Madejski) [reply tutorials-examples/portal-4.rst:1]: fixed!

[the remark this answers is not in the document]
Comment (Madejski) [reply tutorials-examples/portal-4.rst:1]: Very helpful!  I wasn't clear there..

[the remark this answers is not in the document]
Comment (Madejski) [reply tutorials-examples/portal-4.rst:1]: very helpful - I wasn't clear there.

Comment (Madejski) [reply tutorials-examples/portal-4.rst:1]: OK

Comment (Madejski) [reply tutorials-examples/portal-4.rst:1]: done

Comment (Madejski) [reply tutorials-examples/portal-4.rst:1]: OK

**Review by MelissaGraham — APPROVED** [review]
Made a bunch of final commits to get through all requested changes, will leave record in the Jira ticket.

**Pull request merged**